core/src/com/fallenflame/game/GDXRoot.java

```diff
@@ -105,13 +109,21 @@ public void exitScreen(Screen screen, int exitCode) {
 		if (screen == loading) {
 			//Still finish loading everything before shutting down
 			engine.loadContent();
+			Gdx.input.setInputProcessor(levelSelect);
```

Comment:
I'm testing it now, but I don't think we need any of this, as that's the purpose of the multiplexer.

Reply:
Nevermind that makes no sense, but it did not work.